src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTryWithResourceAloneSingle.java

```diff
@@ -31,9 +31,6 @@ void test() throws IOException {
         try (BufferedReader br1 = new BufferedReader(null);
                 BufferedReader br2 = new BufferedReader(br1)) { ; }
         catch (IOException e) { ; }
-        try (BufferedReader br1 = new BufferedReader(null);
-                // violation below ''}' at column 64 should be alone on a line'
-                BufferedReader br2 = new BufferedReader(br1)) {} catch (IOException e) { ; }
```

Comment:
Is this supposed to be a violation and not treated as one block in a single line? because after my changes the check detects this as a single block on a line so, no violation. all the similar cases for other tokens are not detected as violation also examples:

```
PS D:\test> cat config.xml
<?xml version="1.0"?>
    <property name="charset" value="UTF-8"/>

 <module name="TreeWalker">
        <module name="RightCurly">
            <property name="option" value="alone_or_singleline"/>
        </module>
    </module>
</module>
PS D:\test> cat src/Test.java
public class Test {

    public static void main(String[] args) {
        if (true
                && false ) { } else { }

        do {int a = 5;
        int x = 7;
        int m = 3;} while (true);
    }
    public int add (int n
            , int m
            , int x) {  return n + m + x;}
}

PS D:\test> java  -jar checkstyle-10.13.0-all.jar -c config.xml src/Test.java
Starting audit...
Audit done.
PS D:\test> 

```
so if these cases are supposed to be a violation. I can fix it but in this PR? or open new issue

Reply:
This is alone or singleline. The try isn't a single line obviously, but is it alone? If it is alone, then no violation is expected.

> alone 	The brace must be alone on the line.

This does not look to be alone, so a violation is expected IMO.

Reply:
```
private static boolean isBlockAloneOnSingleLine(Details details) {
       .......
        return TokenUtil.areOnSameLine(details.lcurly, details.rcurly)
            && (nextToken == null || !TokenUtil.areOnSameLine(details.rcurly, nextToken)
                || isRightcurlyFollowedBySemicolon(details));
    }
```
this is the code to detect a single block in a line it is checking if the rcurly and lcurly in single line and the rcurly and the next token is not in the same line

so in this case 
```
try( x = 5;
                 y = 6) {} catch {}
                           l r
```
the rcurly and lcurly on same line so single block in a line

Reply:
the reason for this was detected as a violation before because the check wasn't even try to  detect try-catch block as block in a single line but this is not the case now as after this PR the check detect it as a single line so no violation

Reply:
I will open a new issue for this as this problem is not a part of the issue

Reply:
I don't know what I am being pinged in this section for.

Reply:
to review the new issue if it should be approved

Reply:
I don't review issues for approval. That is @romani